aspnetcore/blazor/components/render-modes.md

```diff
@@ -178,44 +178,48 @@ To enable global interactivity when creating a Blazor Web App:
 
 For more information, see <xref:blazor/tooling>.
 
-## Apply a render mode programatically
+## Apply a render mode in code
```

Comment:
Hm, I actually think 'programmatically' might be a better work here, since one could argue that all render modes are applied "in code".

Or maybe 'dynamically' would be a better word, since it implies that the render mode is determined at runtime?

Reply:
Thinking about this more, I don't want the word 'dynamically' to give the illusion that this has anything to do with the auto render mode. Maybe 'programmatically' is better after all. What do you think?